src/modules/jcms_rest/src/JcmsRestServiceProvider.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Drupal\jcms_rest;
+
+use Drupal\Core\DependencyInjection\ContainerBuilder;
+use Drupal\Core\DependencyInjection\ServiceModifierInterface;
+
+/**
+ * Adds application/vnd.elife.annual-report-list+json as a known format.
+ */
+class JcmsRestServiceProvider implements ServiceModifierInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function alter(ContainerBuilder $container) {
+    if ($container->has('http_middleware.negotiation') && is_a($container->getDefinition('http_middleware.negotiation')->getClass(), '\Drupal\Core\StackMiddleware\NegotiationMiddleware', TRUE)) {
+      $container->getDefinition('http_middleware.negotiation')->addMethodCall('registerFormat', ['jcms_json', ['application/vnd.elife.annual-report-list+json']]);
```

Comment:
there is also a `;version=1` suffix in the `Accept` header, which is sent by journal, so that it's actually `application/vnd.elife.annual-report-list+json;version=1`. However, this class seems to be not particularly tied to any of the APIs, while this header changes depending on which API is being called.